packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FGMGroundOverlayController.h

```diff
@@ -0,0 +1,64 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <CoreLocation/CoreLocation.h>
+#import <Flutter/Flutter.h>
+#import <Foundation/Foundation.h>
+#import <GoogleMaps/GoogleMaps.h>
+#import <UIKit/UIKit.h>
+
+#import "messages.g.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// Controller of a single ground overlay  on the map.
+@interface FGMGroundOverlayController : NSObject
+
+/// The ground overlay this controller handles.
+@property(strong, nonatomic) GMSGroundOverlay *groundOverlay;
+
+/// Whether ground overlay is created with bounds or position.
+@property(nonatomic, assign, getter=isCreatedWithBounds) BOOL createdWithBounds;
+
+/// Zoom level when ground overlay is initialized with position.
+@property(nonatomic, strong, nullable) NSNumber *zoomLevel;
+
+/// Initializes an instance of this class with a GMSGroundOverlay, a map view, and identifier.
+- (instancetype _Nullable)initWithGroundOverlay:(GMSGroundOverlay *)groundOverlay
```

Comment:
Does `nullable instancetype` not work? (I can never remember the exact set of cases where `nullable` doesn't work).

Reply:
removed nullable annotation, as the function never returns null instance

Reply:
I removed the nullable annotation from these instance initialisations as they do not explicitly return nil.
There isn't any handling for null instances on code either.
I would assume that `self = [super init];` returns instance, and if not, something is really broken.